.github/base.mega-linter.yaml

```diff
@@ -29,6 +29,7 @@ PYTHON_MYPY_CONFIG_FILE: LINTER_DEFAULT
 PYTHON_PYLINT_CONFIG_FILE: LINTER_DEFAULT
 PYTHON_RUFF_CONFIG_FILE: LINTER_DEFAULT
 REPOSITORY_CHECKOV_CONFIG_FILE: LINTER_DEFAULT
+REPOSITORY_GITLEAKS_CONFIG_FILE: LINTER_DEFAULT
```

Comment:
It doesn't look like this is needed since [the default is already `.gitleaks.toml`](https://megalinter.io/latest/descriptors/repository_gitleaks/).

Reply:
That's what I thought, but it didn't work for me locally when I tried it without that. I'll try again--maybe I mis-spelled it or something silly like that.

Reply:
Pretty sure I must have done something dumb, as suspected. I just ran took away the `REPOSITORY_GITLEAKS_CONFIG_FILE: LINTER_DEFAULT` and Gitleaks ran (it showed up as successfully running on the linter logs) and behaved as expected.